layouts/auth.vue

```diff
@@ -32,39 +32,36 @@
 </template>
 
 <script lang="ts">
```

Comment:
can we use <script setup> for all components  ?

Reply:
as we discussed, sugar will come later